docs/build/tutorials/platform/deploying-cross-chain-evm-bridge.md

```diff
@@ -0,0 +1,324 @@
+# Deploying Cross-Chain EVM <-> EVM Bridge
+
+## Introduction
+
+In this tutorial, we will be building a bridge between **WAGMI** and **Fuji**. This bridge will help us to transfer native **WGM** coin wrapped into **wWGM** back and forth from the WAGMI chain to the Fuji chain. Using this guide, you can deploy a bridge between any EVM-based chains for any ERC20 tokens.
+
+The wrapped version of a native coin is its pegged ERC20 representation. Wrapping it with the ERC20 standard makes certain processes like delegated transactions much easier. You can easily get wrapped tokens by sending the native coin to the wrapped token contract address.
+
+> WAGMI is an independent EVM-based test chain deployed on a custom subnet on the Avalanche network.
+
+We will be using **Chainsafe**'s bridge repository, to easily set up a robust and secure bridge.
+
+## Workflow of the Bridge
+
+Two chains are not interconnected. But we could communicate between the chains through off-chain relayers. Relayers watch for events (by polling blocks) on one chain and perform necessary action using those events on the other chain. This way we can also perform bridging of tokens from one chain to the other chain through the use of smart contracts.
+
+Here is the basic high-level workflow of the bridge -
+
+* User deposits token on the Bridge contract
+* Bridge contract asks Handler contract to perform deposit action
+* Handler contract **locks** the deposited token in the token safe
+* Bridge contract emits `Deposit` event
+* Relayer receives the `Deposit` event from the source chain
+* Relayer creates a voting proposal on the destination chain to mint a new token
+* After threshold relayer votes, the proposal is executed
+* Tokens are **minted** to the recipient's address
+
+Bridging tokens from source to destination chain involves the **lock and mint** approach. Whereas bridging tokens from destination to source chain involves **burn and release** approach. We cannot mint and burn tokens that we do not control. Therefore we lock them in the token safe on the source chain. And mint the corresponding token (which we will deploy and hence control) on the destination chain.
+
+![](/img/chainsafe-bridge-1-workflow.png)
+
+## Requirements
+
+These are the requirement to follow this tutorial -
+
+* Add WAGMI and Fuji chain on the MetaMask network
```

Comment:
Link to WAGMI (https://0bb4a0f8.avalanche-docs.pages.dev/build/tutorials/platform/subnets/wagmi#adding-wagmi-to-metamask) and Fuji (https://0bb4a0f8.avalanche-docs.pages.dev/build/tutorials/platform/fuji-workflow#set-up-fuji-network-on-metamask-optional) using relative path, not direct URL.